xmake/modules/core/tools/dmd.lua

```diff
@@ -83,6 +83,8 @@ function nf_symbol(self, level)
             _g.symbol_maps = maps
         end
         return maps[level .. '_' .. kind] or maps[level]
+    elseif kind == "dcld" then
+        return "-g"
```

Comment:
if should be only for windows (pdb) and you need add `if level == "debug" ` or use map `{debug = "-g"}`

```
function nf_symbol(self, level, target)
    local kind = self:kind()
    if language.sourcekinds()[kind] then
        local maps = _g.symbol_maps
        if not maps then
            maps = {
                debug  = {"-g", "-debug"}
            }
            _g.symbol_maps = maps
        end
        return maps[level .. '_' .. kind] or maps[level]
    elseif (kind == "dcld" or kind == "dcsh") and target:is_plat("windows") and level == "debug" then
        return "-g"
    end
end

```